spec/parallel_tests/test/runtime_logger_spec.rb

```diff
@@ -52,7 +52,7 @@ def test_foo
     skip if RUBY_PLATFORM == "java" # just too slow ...
     Dir.chdir(Bundler.root.join("spec/fixtures/minitest4")) do
       Bundler.with_clean_env do
-        sh "bundle --local --quiet"
+        sh "bundle --quiet"
```

Comment:
should be `bundle check || bundle` ?
also minitest 4 is kinda dead too ... so maybe time to drop that, that was mostly rails 3 support

Reply:
Yeah! :+1: